### PR TITLE
Varun/rov 12

### PR DESCRIPTION
**Pull request by Varuno8**
please review my code @rachittyagi

### CODE REVIEW

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    <section id="contact" class="py-5">
+        <!-- Contact Form using Bootstrap -->
+        <div class="container form-container">
+            <h2 style="text-align: center; margin-bottom: 70px; font-size: 50px; color: #2c3d50 !important;">
+                CONTACT ME
+                <br />
+                <img src="/Users/varuntyagi/Downloads/WhatsApp Image 2023-09-17 at 20.20.54.jpeg" style="width: 40%;">
+            </h2>
+
+            <form>
```
[the remark this answers is not in the document]
Comment (Varuno8) [reply contact.html]: updated the code

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            text-align: center;
+            margin-bottom: 70px;
+            font-size: 50px;
+            color: #2c3d50 !important;
+          "
+        >
+          CONTACT ME
+          <br />
+          <img
+            src="/Users/varuntyagi/Downloads/WhatsApp Image 2023-09-17 at 20.20.54.jpeg"
```
Comment (yaneshtyagi) [inline contact.html:27]: Do not use absolute path for the assets (like images). Instead use relative path.

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+              required
+            />
+          </div>
+          <div class="form-group">
+            <label for="phone">Phone Number</label>
+            <input
+              type="tel"
+              class="form-control"
+              id="phone"
+              placeholder="Phone Number"
```
Comment (yaneshtyagi) [inline contact.html:59]: Inconsistent grammar. Use sentence case like other fields for placeholder text.

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    <script>
+      function validateForm() {
+        // Get references to form fields
+        var fullName = document.getElementById("fullName").value;
+        var email = document.getElementById("email").value;
+        var phone = document.getElementById("phone").value;
+        var message = document.getElementById("message").value;
+
+        // Check if all fields have valid values
+        if (fullName && email && phone && message) {
```
Comment (yaneshtyagi) [inline contact.html:100]: If a user enters a bunch of spaces (e.g. "      ") in full name or message, this validation will fail. Trim these fields to remove leading and trailing spaces before checking if these have any value.

**Review by yaneshtyagi — CHANGES_REQUESTED** [review]

File: contact.html (hunk 2 above)
Comment (Varuno8) [reply contact.html:27]: I will update the code ASAP

Comment (Varuno8) [reply contact.html:27]: I understand, and I will correct this mistake.

File: contact.html (hunk 3 above)
Comment (yaneshtyagi) [reply contact.html:59]: "Phone Number" change is not resolved.

File: contact.html (hunk 4 above)
Comment (yaneshtyagi) [reply contact.html:100]: The condition is not updated.

**Review by yaneshtyagi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**